test/IDE/complete_ambiguous.swift

```diff
@@ -81,10 +29,12 @@ func arrayWrapper<T>(a: T) -> [T]
 arrayWrapper(overloadedReturn()).#^SKIP_DUPLICATES^#
 
 // SKIP_DUPLICATES: Begin completions
-// SKIP_DUPLICATES_PROPERTY: Decl[InstanceVar]/CurrNominal/IsSystem: count[#Int#]{{; name=.+$}}
-// SKIP_DUPLICATES_PROPERTY-NOT: count[#Int#]
-// SKIP_DUPLICATES_METHOD: Decl[InstanceMethod]/Super/IsSystem: formIndex({#(i): &Int#}, {#offsetBy: Int#})[#Void#]{{; name=.+$}}
-// SKIP_DUPLICATES_METHOD-NOT: formIndex({#(i): &Int#}, {#offsetBy: Int#})[#Void#]
+// SKIP_DUPLICATES-NOT: count[#Int#]
+// SKIP_DUPLICATES-NOT: formIndex({#(i): &Int#}, {#offsetBy: Int#})[#Void#]
+// SKIP_DUPLICATES: Decl[InstanceVar]/CurrNominal/IsSystem: count[#Int#]{{; name=.+$}}
+// SKIP_DUPLICATES: Decl[InstanceMethod]/Super/IsSystem: formIndex({#(i): &Int#}, {#offsetBy: Int#})[#Void#]{{; name=.+$}}
+// SKIP_DUPLICATES-NOT: count[#Int#]
+// SKIP_DUPLICATES-NOT: formIndex({#(i): &Int#}, {#offsetBy: Int#})[#Void#]
```

Comment:
Previously
```
// RUN: %swift-ide-test -code-completion  -source-filename %s -code-completion-token=SKIP_DUPLICATES > %t
// RUN: %FileCheck %s --input-file %t --check-prefixes=SKIP_DUPLICATES,SKIP_DUPLICATES_PROPERTY
// RUN: %FileCheck %s --input-file %t --check-prefixes=SKIP_DUPLICATES,SKIP_DUPLICATES_METHOD
```
We can do this by `#^SKIP_DUPLICATES?check=SKIP_DUPLICATES,SKIP_DUPLICATES_PROPERTY;check=SKIP_DUPLICATES,SKIP_DUPLICATES_METHOD^#` But it's not worth it.